Integrating into Angular CLI v1.4.5

Seems that the Angular CLI only accepts dropping the file extensions `.ts` and `.js`

The line that causes the build issue is [here](https://github.com/segmentio/analytics-node/blob/4290503572a7c9661661dbcafef75b573cdbda0b/index.js#L9).

A simple fix would be to change this `const version = require('./package').version` to this `const version = require('./package.json').version`.

I know this is outside the scope of this library but thought you all may want to offer Angular CLI support as well :)

### Console Error
```
ERROR in ./node_modules/analytics-node/index.js
Module not found: Error: Can't resolve './package' in '/.../node_modules/analytics-node'
resolve './package' in '/.../node_modules/analytics-node'
  using description file: /.../node_modules/analytics-node/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /.../node_modules/analytics-node/package.json (relative path: .)
    using description file: /.../node_modules/analytics-node/package.json (relative path: ./package)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /.../node_modules/analytics-node/package doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /.../node_modules/analytics-node/package.ts doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /.../node_modules/analytics-node/package.js doesn't exist
      as directory
        /.../node_modules/analytics-node/package doesn't exist
[/.../node_modules/analytics-node/package]
[/.../node_modules/analytics-node/package.ts]
[/.../node_modules/analytics-node/package.js]
[/.../node_modules/analytics-node/package]
 @ ./node_modules/analytics-node/index.js 9:16-36
 @ ./src/app/store/effects/user.effects.ts
 @ ./src/app/store/effects/index.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi webpack-dev-server/client?http://localhost:4200 ./src/main.ts

webpack: Failed to compile.
```

--- a/index.js
+++ b/index.js
@@ -6,7 +6,7 @@ const validate = require('@segment/loosely-validate-event')
 const request = require('superagent')
 const debug = require('debug')('analytics-node')
 const uid = require('crypto-token')
-const version = require('./package').version
+const version = require('./package.json').version
 
 const setImmediate = global.setImmediate || process.nextTick.bind(process)
 const noop = () => {}

